src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -125,9 +125,9 @@ public void testLoginAsyncFailure() throws SaltStackException {
     public void testRunRequest() throws SaltStackException {
         stubFor(post(urlEqualTo("/run"))
                 .willReturn(aResponse()
-                    .withStatus(HttpURLConnection.HTTP_OK)
-                    .withHeader("Content-Type", "application/json")
-                    .withBody(JSON_RUN_RESPONSE)));
+                        .withStatus(HttpURLConnection.HTTP_OK)
```

Comment:
Nitpick: it seems like you changed spacing here. Again please make sure that your patches only change lines that are supposed to be changed.